marxs/source/labSource.py

```diff
@@ -105,3 +107,77 @@ def generate_photons(self, exposuretime):
         photons.add_column(Column(name='dir', data=dir.T))
         photons.add_column(Column(name='polarization', data=polarization_vectors(dir.T, photons['polangle'])))
         return photons
+
+class LabPointSourceCone(Source):
+    '''In-lab point source
+
+    - Photons are uniformly distributed in all directions in cone (from the point). Cone is meant to refer to the volume swept out by a solid angle in a sphere.
+    - Photon start position is source position (center of the sphere = tip of cone)
+
+    Parameters
+    ----------
+    position: 3 element list
+        3D coordinates of photon source
+    direction: array
```

Comment:
Please order the explanation of direction, delta to match the order of the parameters of the `__init__` function